ext/capn_proto/dynamic_struct_builder.cc

```diff
@@ -17,7 +17,8 @@ namespace ruby_capn_proto {
     ClassBuilder("DynamicStructBuilder", rb_cObject).
       defineAlloc(&alloc).
       defineMethod("which", &which).
-      defineMethod("write", &write).
+      defineMethod("write", &writePacked).
+      defineMethod("writePacked", &writePacked).
```

Comment:
Ruby method should be 🐍 case

Reply:
updated to both be snake case. Rest of the (gem) code base uses snake case for method names as well.